subsys/net/ip/net_shell.c

```diff
@@ -853,6 +853,16 @@ static void print_eth_stats(struct net_if *iface, struct net_stats_eth *data,
 	PR("Mcast received   : %u\n", data->multicast.rx);
 	PR("Mcast sent       : %u\n", data->multicast.tx);
 
+	PR("Send errors      : %u\n", data->errors.tx);
+	PR("Receive errors   : %u\n", data->errors.rx);
+	PR("Collisions       : %u\n", data->collisions);
+	PR("Send Drops       : %u\n", data->tx_dropped);
+	PR("Send timeouts	 : %u\n", data->tx_timeout_count);
```

Comment:
Alignment looks wrong, perhaps there is \t vs space in the string. There should not be \t chars in these printable string.

Reply:
Thanks, fixed it. There was a tab.